packages/@n8n/task-runner-python/src/health_check_server.py

```diff
@@ -20,8 +20,9 @@ async def start(self, config: HealthCheckConfig) -> None:
             self.server = await asyncio.start_server(
                 self._handle_request, config.host, config.port
             )
+            actual_port = self.server.sockets[0].getsockname()[1] # for OS-assigned port in tests 
             self.logger.info(
-                f"Health check server listening on {config.host}, port {config.port}"
+                f"Health check server listening on {config.host}, port {actual_port}"
```

Comment:
Did we have a way to control the loglevel of this, is there a risk that this logging happens on loglevel above `info` and we never log this -> it never ends up on `stdout_buffer` -> tests fail at `_wait_for_health_check_port` ? 

I think there's no such risk here, we only monitor the port logline at tests and they're run with level where this gets printed?